lib/js/app/modules/queries/reducer.ts

```diff
@@ -20,6 +21,7 @@ export const initialState: ReducerState = {
   isSavingQuery: false,
   saved: [],
   isLimited: false,
+  limitReached: false,
```

Comment:
Right now there is `isLimited ` -> which indicates the cache possibilities and `limitReached `. For external developer this names are not self-descriptive. 

Please rename `limitReached` -> `queriesExecutionLimitReached `. For caching user story we will also rename `isLimited` to something more meaningful.